examples/se3_sam.cpp

```diff
@@ -349,7 +351,7 @@ int main()
 
     //// Simulator ###################################################################
     poses_simu. push_back(X_simu);
-    poses.      push_back(Xi + SE3Tangentd::Random());  // use very noisy priors
+    poses.      push_back(Xi + (SE3Tangentd::Random()*0.1));  // use very noisy priors
```

Comment:
remove the "very noise priors" comment if using `0.1*rand`

Reply:
Fix : `// use noisy priors`.